modules/applications/pages/ui-builder/bonita-ui-builder.adoc

```diff
@@ -32,7 +32,9 @@ They are not standalone elements (such as pages and forms) in a process anymore,
 
 Bonita also comes with four xref:runtime:bonita-applications-interface-overview.adoc[Bonita Applications].
 
-“Applications” are very similar to “Living Applications”, which are created with Bonita UI Designer and will be phased out. Once pushed to production, Applications cannot be live edited.
+“Applications” are very similar to “Living Applications”, which are created with Bonita UI Designer.
```

Comment:
Why do we remove the fact that the application cannot be live edited ?

Reply:
Because of the 'compatibility with platform mode' project https://bonitasoft.atlassian.net/wiki/spaces/SF/pages/23332225146/Deploying+applicationLink+in+platform+mode+-+Product+Requirement+Document#User-flow

Reply:
ok, the link can be updated, but not the application itself : we cannot change the application content once deployed.

Reply:
```suggestion
“Applications” are very similar to “Living Applications”, which are created with Bonita UI Designer. Once pushed to production, the Application's content cannot be edited.
```